.circleci/config.yml

```diff
@@ -49,6 +49,26 @@ commands:
       - run: git branch -f stable HEAD
       - run: git push origin stable
 
+  build_and_deploy_docs_to_github_pages:
+    description: Build Sphinx documentation and push GitHub stable branch which points to the latest relased tag/commit
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "b6:94:61:c3:e1:8f:25:e5:43:6d:51:9e:43:c4:16:11"
+      - common_setup
+      - checkout
```

Comment:
Why do we need this? If I read things correctly, checkout is part of the common_setup step.

Reply:
We don't need it so this has been removed.